stripper/ze_doom3_v2.cfg

```diff
@@ -156,4 +137,560 @@ modify:
+	"targetname" "strp_stg3_tp_avoid_3"
+	"classname" "trigger_teleport"
+	"origin" "-3303 13052.1 -3903.08"
+	"spawnflags" "1"
+	"startdisabled" "1"
+	"target" "hellteleport"
+	"UseLandmarkAngles" "1"
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"hammerid" "2673"
+	}
+	insert:
+	{
+		"OnStartTouch" "strp_stg3_tp_avoidEnable101"
+		"OnStartTouch" "strp_stg3_tp_avoid_2Enable101"
+		"OnStartTouch" "strp_stg3_tp_avoid_3Enable101"
+	}
+}
+; Add tp to stage 2 that exists on css 
+add:
+{
+	"targetname" "strp_tp_stg2_gap"
+	"classname" "trigger_teleport"
+	"origin" "-5502 14562 -4044.5"
+	"spawnflags" "1"
+	"startdisabled" "1"
+	"target" "teler1"
+	"UseLandmarkAngles" "1"
+}
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"hammerid" "2786"
+	}
+	insert:
+	{
+		"onpressed" "strp_tp_stg2_gapEnable321"
+	}
+}
+; Nerf zm spider fireball (This nerf most likely needs tweaking)
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "hurtimp1"
+	}
+	replace:
+	{
+		"damage" "35"
+	}
+}
+; Nerf zm skeleton firebomb (This nerf most likely needs tweaking)
+modify:
+{
+	match:
+	{
+		"classname" "env_explosion"
+		"targetname" "gila3"
+	}
+	replace:
+	{
+		"iMagnitude" "30"
+		"iRadiusOverride" "350"
+	}
+}
+; Fix bfg math counter to reflect model (7 uses instead of 8)
+modify:
+{
+	match:
+	{
+		"classname" "math_counter"
+		"targetname" "bfg_model_counter"
+	}
+	delete:
+	{
+		"OnHitMax" "item_bfg_launcher_modelKill0-1"
+	}
+	replace:
+	{
+		"max" "7"
+	}
+	insert:
+	{
+		"OnHitMax" "item_bfg_launcher_modelKill2.3-1"
+	}
+}
+; Move heart item freeze more to center 
+modify:
+{
+	match:
+	{
+		"targetname" "heart_template_freeze"
+		"classname" "point_template"
+	}
+	replace:
+	{
+		"origin" "-6699 14253 -3883"
+	}
+}
+; Move electro nade more to center
+modify:
+{
+	match:
+	{
+		"targetname" "grenade_spawn"
+		"classname" "point_template"
+	}
+	replace:
+	{
+		"origin" "-6314 14087 -4141"
+	}
+}
+; Add blocking damage to stg 2 elevator
+modify:
+{
+	match:
+	{
+		"targetname" "stage2_door1"
+		"classname" "func_door"
+	}
+	replace:
+	{
+		"dmg" "99999"
+	}
+}
+modify:
+{
+	match:
+	{
+		"targetname" "stage2_doo2"
+		"classname" "func_door"
+	}
+	replace:
+	{
+		"dmg" "99999"
+	}
+}
+; Move where boss hp is added for stage 3 boss fight (prevents cheesing the boss where 1 ct goes in)
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "Noctali_Boss_Hp_To_Boss1"
+	}
+	delete:
+	{
+		"OnStartTouch" "hp_counters_templalteForceSpawn01"
+		"OnStartTouch" "pirate_counterAdd320.5-1"
+		"OnStartTouch" "pirate_hp_backupAdd320.5-1"
+	}
+}
+add:
+{
+	"model" "*334"
+	"classname" "trigger_multiple"
+	"spawnflags" "1"
+	"origin" "-11525 -8945.5 -262"
+	"startdisabled" "0"
+	"wait" "60"
+	"filtername" "Filter_Team_Humans"
+	"OnStartTouch" "hp_counters_templalteForceSpawn01"
+	"OnStartTouch" "pirate_counterAdd320.5-1"
+	"OnStartTouch" "pirate_hp_backupAdd320.5-1"
+}
```

Comment:
This will be abusable since the original Noctali_Boss_Hp_To_Boss1 used a push to prevent people from triggering it multiple times, but this new trigger_multiple can be left and re-entered multiple times to keep scaling up the boss.

You should instead be trying a fix that doesn't allow humans to be outside the arena when the fight starts (all have gone through Noctali_Boss_Hp_To_Boss1 or get killed), as this current fix also won't stop the boss from leaving the arena to go after players outside of it.